packages/shared/src/node-apis/redis.ts

```diff
@@ -74,11 +74,15 @@ export default class RedisClient {
     return value ? broadcastParsers[chain].parse(JSON.parse(value)) : null;
   }
 
-  async getDeposits(chain: Chain, address: string) {
+  async getDeposits(chain: Chain, asset: Asset, address: string) {
     const parsedAddress = chain === 'Polkadot' ? ss58ToHex(address) : address;
     const key = `deposit:${chain}:${parsedAddress}`;
     const value = await this.client.get(key);
-    return value ? deposits.parse(JSON.parse(value)) : [];
+    return value
```

Comment:
is the order of this list guaranteed? might make sense to sort it by `deposit_chain_block_height`

Reply:
done